pkg/cache/context.go

```diff
@@ -873,19 +871,19 @@ func (ctx *Context) StartPodAllocation(podKey string, nodeID string) bool {
 	return ctx.schedulerCache.StartPodAllocation(podKey, nodeID)
 }
 
-func (ctx *Context) notifyTaskComplete(appID, taskID string) {
-	log.Log(log.ShimContext).Debug("NotifyTaskComplete",
-		zap.String("appID", appID),
-		zap.String("taskID", taskID))
-	if app := ctx.GetApplication(appID); app != nil {
-		log.Log(log.ShimContext).Debug("release allocation",
-			zap.String("appID", appID),
+func (ctx *Context) notifyTaskComplete(app *Application, taskID string) {
+	if app == nil {
+		log.Log(log.ShimContext).Debug("In notifyTaskComplete but app is nil",
 			zap.String("taskID", taskID))
-		ev := NewSimpleTaskEvent(appID, taskID, CompleteTask)
-		dispatcher.Dispatch(ev)
-		if app.GetApplicationState() == ApplicationStates().Resuming {
-			dispatcher.Dispatch(NewSimpleApplicationEvent(appID, AppTaskCompleted))
-		}
+		return
+	}
+	log.Log(log.ShimContext).Debug("notifyTaskComplete and release allocation",
```

Comment:
Maybe we can keep origin message? `release allocation`

Reply:
I change the message to "notifyTaskComplete and release allocation" because I combine 2 logs to make it only log once. Perhaps I can change it to " release allocation in notifyTaskComplete"?

Reply:
Got it. It is fine to keep current message :)